File: cs/Iris/ParserUtils/LexParseException/DefaultCaseException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class DefaultCaseException : BaseException
    {
        public DefaultCaseException(int lNum, int cNum)
            : base(lNum, cNum)
        {

        }

        public override string ExceptionToString()
        {
            StringBuilder str = new StringBuilder();

            str.Append(base.stringTrace());
            str.Append("default case already exists");

            return str.ToString();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/DefaultCaseException.java
package com.rokuan.iris.exception;

public class DefaultCaseException extends BaseException
{
	public DefaultCaseException(int lNum, int cNum)
	{
		super(lNum, cNum);
	}

	public String ExceptionToString()
	{
		StringBuilder str = new StringBuilder();

		str.append(super.stringTrace());
		str.append("default case already exists");

		return str.toString();
	}
}
